README.md

```diff
@@ -253,6 +253,9 @@ Your Space ID.
 #### host (default: `'cdn.contentful.com'`)
 Set the host used to build the request URI's.
 
+#### basePath (default: ``)
```

Comment:
@googol fixed 🙈  - we consider adding tests for this in the next version so it won't happen again :)